02 - Community Expansions/02.01 Plugins by Category/Dictionary and Spellchecking Plugins.md

```diff
@@ -0,0 +1,27 @@
+---
+aliases:
+- 
+tags: 
+- seedling 
+publish: true
+---
+
+
+# Dictionary and Spellchecking Plugins
+
+%% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. %%
+#placeholder/description 
+
+## Plugins in this category
+
+%% Add a bullet here and link to the plugins you'd like to categorize! %%
+
+- [[obsidian-dictionary-plugin|Dictionary]]
+- [[obsidian-languagetool-plugin|LanguageTool Integration]]
+- [[obsidian-orthography|Obsidian Orthography]]
```

Comment:
you can add `- [[obsidian-vale|Vale]]` here

Reply:
Done!